go-packages/meep-metric-store/metric-store.go

```diff
@@ -266,11 +266,20 @@ func (ms *MetricStore) GetInfluxMetric(metric string, tags map[string]string, fi
 	// Tags
 	tagStr := ""
 	for k, v := range tags {
+		mv := strings.Split(v, ",")
```

Comment:
I recommend adding a UT for this new functionality.

Reply:
added ut test for 2 loggers in httplog-test